packages/pluggableWidgets/progress-bar-web/src/ProgressBar.tsx

```diff
@@ -0,0 +1,45 @@
+import { createElement, FunctionComponent, useCallback } from "react";
+import { ProgressBarContainerProps } from "../typings/ProgressBarProps";
+import { ProgressBar as ProgressBarComponent } from "./components/ProgressBar";
+import { defaultValues, ProgressBarValues } from "./progressBarValues";
+
+export const ProgressBar: FunctionComponent<ProgressBarContainerProps> = props => {
+    function getProgressBarValues(): ProgressBarValues {
+        switch (props.type) {
+            case "dynamic":
+                return {
+                    currentValue: Number(props.dynamicCurrentValue?.value ?? defaultValues.currentValue),
```

Comment:
I already check for this in `ProgressBar.editorConfig.jsx` at compile time. Should I add another runtime check and show an `Alert` if they are not set? (Also for the expression case)

Reply:
Sorry.. It was supposed to be `should`, sometimes the values are not available then if we set defaults it will just change during runtime. Better fallback as 0

Reply:
Will change, but please not that setting the current value to 0 is not the same as setting the progress to 0%, since it depends on the minimum and maximum value.

Reply:
In this case we need to set as 0% instead of 50

Reply:
@diego-antonelli 👆  just to make sure it wasn't missed

Reply:
No, I checked the code and looks fine